include/click/dpdkdevice.hh

```diff
@@ -1,6 +1,23 @@
 #ifndef CLICK_DPDKDEVICE_HH
 #define CLICK_DPDKDEVICE_HH
 
+//Prevent bug under some configurations (like travis-ci's one) where these macros get undefined
+#ifndef UINT8_MAX
```

Comment:
It looks like they are #define'd in <stdint.h>. I will make a commit to glue.hh to include that header at userlevel. Then maybe these weird defines won't be required?